mnist_batch_norm/mnist_batch_norm.cpp

```diff
@@ -144,8 +144,16 @@ int main()
               ens::PrintLoss(),
               ens::ProgressBar(),
               // Stop the training using Early Stop at min loss.
-              ens::EarlyStopAtMinLoss());
-
+              ens::EarlyStopAtMinLoss<arma::mat>(
+                  [&](const arma::mat& /* param */)
+                  {
+                    double validationLoss =
+                    model.Evaluate(validX, validY);
+                    std::cout << "Validation loss: "
+                              << validationLoss
+                              << std::endl;
```

Comment:
```suggestion
                        model.Evaluate(validX, validY);
                        std::cout << "Validation loss: "
                                  << validationLoss
                                  << std::endl;
```

Hopefully the idendation is correct (tab twice - 4 spaces).

Reply:
I think the 4 spaces are special lambda right?

Reply:
I see because I have just cut the lines